nvflare/private/fed/server/admin.py

```diff
@@ -274,9 +278,10 @@ def send_requests(self, requests: dict, timeout_secs=2.0, optional=False) -> [Cl
         """
 
         for _, request in requests.items():
-            with self.sai.new_context() as fl_ctx:
-                self.sai.fire_event(EventType.BEFORE_SEND_ADMIN_COMMAND, fl_ctx)
-                request.set_header(ReservedHeaderKey.PEER_PROPS, copy.deepcopy(fl_ctx.get_all_public_props()))
+            # with self.sai.new_context() as fl_ctx:
+            self.sai.fire_event(EventType.BEFORE_SEND_ADMIN_COMMAND, fl_ctx)
+            shared_fl_ctx = gen_new_peer_ctx(fl_ctx)
```

Comment:
what do we gain when we re-use this fl_ctx?

Reply:
there are several non-sticky props need to be used. new FLContext does not contain them.